code/datums/action.dm

```diff
@@ -139,13 +251,13 @@
 
 
 /datum/action/xeno_action/activable
-	var/ability_name
+	var/use_plasma_on_activate
```

Comment:
This doesn't seem to be modified anywhere, and it's a flag now.

Reply:
yeah

Reply:
nothing seems to rely on the proc using plasma so i'll just remove that part.